--- kong-2.0.4-0.rockspec	2020-09-08 14:52:00.257090462 +0100
+++ kong-2.0.4-0.rockspec	2020-09-10 10:14:17.960907669 +0100
@@ -35,7 +35,7 @@
   "lua-resty-cookie == 0.1.0",
   "lua-resty-mlcache == 2.4.1",
   "lua-messagepack == 0.5.2",
-  "lua-resty-openssl == 0.4.4",
+  "lua-resty-openssl == 0.6.3",
   "lua-resty-counter == 0.2.0",
   -- external Kong plugins
   "kong-plugin-azure-functions ~> 0.4",
